package/suse-migration-services-spec-template

```diff
@@ -89,6 +89,10 @@ install -D -m 644 systemd/suse-migration-ssh-keys.service \
 install -D -m 644 systemd/suse-migration-console-log.service \
     %{buildroot}%{_unitdir}/suse-migration-console-log.service
 
+
+%post
+suse-migration-pre-checks
+
```

Comment:
The %post of the suse-migration-services should not run the pre-checks, only the %post of the activation package should do it. Remember only the activation package is installed on the host to upgrade. Only there it makes sense to run pre-checks

Reply:
apart from the above the spec changes looks now really good

Reply:
Done